routes/v1.js

```diff
@@ -46,99 +93,131 @@ api.declare({
   deferAuth:      true,
   scopes:         ['aws-provisioner:create-worker-type:<workerType>'],
   input:          SCHEMA_PREFIX_CONST + 'create-worker-type-request.json#',
-  output:         SCHEMA_PREFIX_CONST + 'create-worker-type-response.json#',
+  output:         SCHEMA_PREFIX_CONST + 'get-worker-type-response.json#',
   title:          "Create new Worker Type",
   description: [
-    // Document what this method does
+    "Create a worker type and ensure that all EC2 regions have the required ",
+    "KeyPair"
   ].join('\n')
-}, function(req, res) {
+}, 
+function(req, res) {
   var ctx         = this;
   var input       = req.body;
   var workerType  = req.params.workerType;
```

Comment:
Done.  I am now asserting that in the WorkerType.create method.  Errors there will bubble up to the response.